### PR TITLE
update TOC doc

### PR DESCRIPTION
**Pull request by herohua**
@superyyrrzz @928PJY @live1206 @fenxu

### CODE REVIEW

File: Documentation/tutorial/intro_toc.md [hunk 1]
```diff
@@ -27,16 +29,25 @@ Basic format
...
-3. URI with `xref` scheme, for example, `xref:System.String`, the value is the `uid` of the file to be [cross-referenced](../spec/docfx_flavored_markdown.md#cross-reference).
+3. URI with `xref` scheme or `@` syntax, for example, `xref:System.String` or `@System.String`. The value is the `uid` of the file to [cross-reference](../spec/docfx_flavored_markdown.md#cross-reference).
 
 ### YAML format TOC `toc.yml`
 
+Comparing to `toc.md`, `toc.yml` represents a structured data model and conforms to the [YAML standard](http://www.yaml.org/spec/1.2/spec.html). It supports advanced functionalities.
+
+#### Data model for `toc.yml`
+
+The data model for `toc.yml` is either an **array** of *TOC Item Object*s, or an **object** consisting of two properties - "metadata": an **object** containing all metadata of the TOC, and "items": an **array** of *TOC Item Object*s.
```
Comment (superyyrrzz) [inline Documentation/tutorial/intro_toc.md]: >metadata [](start = 124, length = 8)

metadata is an extensible member in toc.yml, so no need for a key named "metadata"
[https://github.com/dotnet/docfx/blob/dev/src/Microsoft.DocAsCode.DataContracts.Common/TocRootViewModel.cs#L23](https://github.com/dotnet/docfx/blob/dev/src/Microsoft.DocAsCode.DataContracts.Common/TocRootViewModel.cs#L23)

File: Documentation/tutorial/intro_toc.md [hunk 2]
```diff
@@ -47,49 +58,64 @@ Three kinds of links are supported:
...
+## *href* in detail
+
+If a *TOC Item* is linking to a relative path, there are three cases:
 
 1. Linking to another *TOC File*, for example, `href: examples/toc.md`.
 2. Linking to a folder, which means, the value of the link ends with `/` explicitly, for example, `href: examples/`
 3. Linking to some local file.
 
-Each case is described in detail below.
+Each case is described in details below.
```
Comment (superyyrrzz) [inline Documentation/tutorial/intro_toc.md]: >in details [](start = 23, length = 10)

in detail?
[https://english.stackexchange.com/questions/23986/in-detail-vs-in-details](https://english.stackexchange.com/questions/23986/in-detail-vs-in-details)

File: Documentation/tutorial/intro_toc.md (hunk 1 above)
Comment (herohua) [reply Documentation/tutorial/intro_toc.md]: I want to follow the same way as OPS

---
In reply to: [208489941](https://github.com/dotnet/docfx/pull/3165#discussion_r208489941) [](ancestors = 208489941)

File: Documentation/tutorial/intro_toc.md (hunk 2 above)
Comment (herohua) [reply Documentation/tutorial/intro_toc.md]: nice

---
In reply to: [208491924](https://github.com/dotnet/docfx/pull/3165#discussion_r208491924) [](ancestors = 208491924)

File: Documentation/tutorial/intro_toc.md (hunk 1 above)
Comment (superyyrrzz) [reply Documentation/tutorial/intro_toc.md]: We can also remove this part, as users will not use it for default template.

**Pull request merged**